BackEnd/src/main/java/com/exadel/carpoolfree/controller/DriveController.java

```diff
@@ -72,4 +109,6 @@ public boolean delete(final @PathVariable Long id) {
         return true;
```

Comment:
I know this has not been changed but it always returns true even if delete not successful, void is fine here